src/tabbar.cpp

```diff
@@ -122,6 +123,21 @@ WebView* TabBar::createNewTab(bool setCurrent)
                 QUITIFNOTCURRENT(webView);
                 emit webActionEnabledChanged(QWebEnginePage::Forward, webView->isWebActionEnabled(QWebEnginePage::Forward));
             });
+    connect(webView->page(), &QWebEnginePage::linkHovered, this, 
+            [=](const QString& url) {
+                auto tabbar = KiwixApp::instance()->getTabWidget();
+                if (url.isEmpty()) {
+                    QToolTip::hideText();
+                } else {
+                    auto link = url;
+                    if (url.startsWith("zim://")) {
+                        link.remove(0, url.indexOf(".zim")+ 4);
+                    }
```

Comment:
Would be better to convert this to a QUrl and use the scheme and path method :
https://doc.qt.io/Qt-5/qurl.html#path